src/browser-driver-manager.js

```diff
@@ -1,259 +1,259 @@
   }
-  // output version installed from npm
-  else if (['version', 'which'].includes(userArgs[0]) && chromedriver) {
-    try {
-      const chromedriverPkg = require('chromedriver');
+}
 
-      if (userArgs[0] === 'version') {
-        console.log(chromedriverPkg.version);
-      } else {
-        console.log(chromedriverPkg.path);
-      }
+/**
+ * Write the environment file to the BDMCacheDir.
+ * @param {object} data - The data to write.
+ * @param {string} data.chromePath - The path to Chrome.
+ * @param {string} data.chromedriverPath - The path to Chromedriver
+ * @param {string} data.version - The version of Chrome and Chromedriver
+ * @throws {Error} - Environment file must be writable.
+ */
+async function setEnv({ chromePath, chromedriverPath, version }) {
+  console.log('Setting env CHROME/CHROMEDRIVER_TEST_PATH/VERSION');
 
-      return;
-    } catch (err) {
-      return error(chalk, 'chromedriver is not installed');
-    }
+  try {
+    await fsPromises.writeFile(
+      path.resolve(getBDMCacheDir(), '.env'),
+      `CHROME_TEST_PATH="${chromePath}"${os.EOL}CHROMEDRIVER_TEST_PATH="${chromedriverPath}"${os.EOL}VERSION="${version}"${os.EOL}`
+    );
+    console.log('CHROME_TEST_PATH is set in', chromePath);
+    console.log('CHROMEDRIVER_TEST_PATH is set in', chromedriverPath);
+    console.log('VERSION:', version);
+  } catch (e) {
+    throw new Error('Error setting CHROME/CHROMEDRIVER_TEST_PATH/VERSION');
   }
+}
 
+/**
+ * Read the environment file from the BDMCacheDir.
+ * @returns {string|undefined} - The content of the environment file.
+ */
+async function getEnv() {
   try {
-    await runBashScript({
-      args: userArgs
-    });
-  } catch (err) {
+    const envPath = path.resolve(getBDMCacheDir(), '.env');
+    const env = await fsPromises.readFile(envPath, 'utf8');
+    return env;
+  } catch (e) {
     return;
   }
 }
 
 /**
- * Output an error message
- * @param {Object} chalk
- * @param {String} msg - Message to display
+ * Log the environment file to the console.
+ * @throws {Error} - Environment file must exist.
  */
-async function error(chalk, msg) {
-  console.error(chalk.red('browser-driver-manager error:'), msg);
-  process.exitCode = 1;
+async function which() {
+  const env = await getEnv();
+  if (!env) {
+    throw new Error('No environment file exists. Please install first');
+  }
+  console.log(env);
 }
 
 /**
- * Output a message for verbose logging
- * @param {Object} chalk
- * @param {Boolean} verbose - If verbose logging is enabled
- * @param {String} msg - Message to display
+ * Read the installed version from the environment file.
+ * @returns {string|null} - The version if one exists.
  */
-async function verboseLog(chalk, verbose, msg) {
-  if (verbose) {
-    console.log(chalk.blue('log:'), msg);
-  }
+async function getVersion() {
+  const pattern = /^VERSION="([\d.]+)"$/m;
+  const env = await getEnv();
+
+  // Search for the pattern in the file path
+  const match = env?.match(pattern);
```

Comment:
This should probably error with the same message as `which` when env doesn't exist.